




export const userSignUp = (req,res ) => {

    try {
        const user = request.body;n
    } catch (error) {
        console.log("error in calling usersigup", error );
    }
}